astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -571,6 +573,7 @@ def test_time_1d_values_time(header_time_1d):
     assert_time_at(header_time_1d, 1, 2450003, 0.1 + 7 / 3600 / 24, 'tai', 'mjd')
 
 
+@pytest.mark.remote_data
```

Comment:
Wut... how did we miss this one? I thought `pytest_remotedata` would throw an error if a test not marked as `remote_data` tries to access the Internet?

Reply:
Yeah somehow it just warned